engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/AnyToDisplayTextNode.java

```diff
@@ -32,6 +33,24 @@ Text showExceptions(
     }
   }
 
+  @Specialization
+  Text convertText(Text self) {
+    final var limit = 80;
+    if (self.length() < limit) {
+      return self;
+    } else {
+      var b = BreakIterator.getWordInstance();
+      b.setText(self.toString());
+      var l = b.preceding(limit);
+      if (l == BreakIterator.DONE) {
+        l = limit;
+      } else {
+        l--;
+      }
+      return Text.create(self.toString().substring(0, l));
```

Comment:
I think I don't understand this logic.

Why are we doing `l--`? It feels like this can get us in the middle of a grapheme cluster, no? If `l` is originally at a boundary position, if we have multi-codepoint cluster before it, my intuition (aided with reading the docs of `BreakIterator::preceding` says that `l--` can put us in the middle of it.

But maybe I don't understand something here. I'd appreciate a comment (in the code) shortly explaining why this `l--` operation is correct.

Reply:
I think it could be much simpler to move the method `take_prefix` from `Text_Utils` to `Core_Text_Utils` and use it here. What do you think?

The less places we have logic for splitting strings on graphemes scattered around, the better.

It has similar functionality and is already tested.

Reply:
`l` points after a space. Hence `l--`.

Reply:
It does, but in `substring(start, end)` the `end` is also supposed to be _after_ the last codepoint we will take right? So IMO it looks like the `--` is unnecessary.

Anyway - I think just best to reuse `take_prefix` - any reasons not to?

Reply:
Using `take_prefix` since 4bc2353